workspace_tools/data_scrape/supported_html.py

```diff
@@ -0,0 +1,107 @@
+HEADER = """
+<!DOCTYPE html>
+<html>
+<head>
+<style>
+
+table, th, td {
+    border: 1px solid black;
+     border-collapse: collapse;
+}
+
+.rotate {
+    height: 140px;
+    white-space: nowrap;
+    width: 30px;
+    position: relative;
+    top: 75px;
+     right: -50px;
+     -moz-transform: rotate(-45.0deg);  /* FF3.5+ */
+      -ms-transform: rotate(-45.0deg);  /* IE9+ */
+       -o-transform: rotate(-45.0deg);  /* Opera 10.5 */
+  -webkit-transform: rotate(-45.0deg);  /* Safari 3.1+, Chrome */
+          transform: rotate(-45.0deg);  /* Standard */
+}
+</style>
+</head>
+<body>
+<div id="users">
+<input class="search" placeholder="Search" />
+<table id="example" class="dataTable display" cellspacing="0" width="80%">
+"""
+
+AVAILABLE = ["NO", "YES"]
+SROW = "<tr>\n"
+EROW = "</tr>\n"
+SBODY = "       <tbody class=\"list\">\n"
+EBODY = "</tbody>\n"
+SHEAD = "<thead>\n"
+EHEAD = "</thead>\n"
+SHEADER1 = "<th><div class=\"rotate\" id=\"filter-"
+SHEADER2 = "\">"
+EHEADER = "</div></th>\n"
+SCELL1 = "          <td class = \""
+SCELL2 = "\">"
+ECELL = "</td>\n"
+
+FOOTER = """</table>
+</div>
+</body>
+"""
+
+SCRIPT1="""
+<script src="https://developer.mbed.org/media/uploads/Sissors/list.js"></script>
```

Comment:
Because first of all, its not mine. Only reason I put it on the mbed site is that modern browsers only accept external scripts which are hosted on https, if the current page it is on is also hosted by https. Since where I got it from is not hosted on https, I had to make a copy. 

I can, and have before, included the script source code in the generated html. Which makes it even larger, it makes much more sense to reference it. 

I don't think I can do much about it: By definition I can't host it somewhere where I can't change it in the future, and making a mirror of someone elses code is one thing, but I don't think it is appropriate to put it here as "mbed code". So if it is a problem you can host that file somewhere else on mbed where I don't have access. Or propose another solution.